src/protocols/aeternity/config.ts

```diff
@@ -33,11 +34,13 @@ export const AE_NETWORK_DEFAULT_SETTINGS: IDefaultNetworkTypeData<IAeNetworkSett
     nodeUrl: AE_NETWORK_MAINNET_NODE_URL,
     middlewareUrl: 'https://mainnet.aeternity.io/mdw',
     backendUrl: 'https://superhero-backend-mainnet.prd.service.aepps.com',
+    tokenSalesUrl: 'https://mainnet.api.dev.tokaen.org/api',
```

Comment:
I don't think that we should add `tokenSalesUrl` into `AE_NETWORK_DEFAULT_SETTINGS`. It should not be a part of an `activeNetwork` defaults.